ImageToSketch/sketchScript.py

```diff
@@ -0,0 +1,84 @@
+from cv2 import cv2
+import os
+import os.path
+from datetime import datetime, date
+
+# To capture image from webcam
+# The image will be stored with the filename as datetime.now() and will be in webcam/data
+
+
+def openCam():
+    datasets = 'webcam'
+    sub_data = 'data'
+    path = os.path.join(datasets, sub_data)
+    if not os.path.isdir(path):
+        os.mkdir(path)
+    webcam = cv2.VideoCapture(0, cv2.CAP_DSHOW)
+    (_, im) = webcam.read()
+    now = str(datetime.now())
+    now = now.replace(':', '')
+    now = now.replace('-', '')
+    now = now.replace('.', '')
+    now = now.replace(' ', '')
+
+    cv2.imwrite(f'webcam\\data\\{now}.png', im)
+    img = f'webcam\\data\\{now}.png'
+    convert(img)
+
+# This is to convert the file passed as parameter to sketch
+
+
+def convert(now):
+    print()
+    img = input('Enter the filename for the output image: ')
+    print()
+    directory = img
+
+    # To create the folder in sketches of that filename
+    # And this folder will contain all images as transition to sketch
+
+    parent_dir = f'C:\\Users\\Hp\\Programming\\Python\\sketch\\sketches'
```

Comment:
This will be more generic:
```suggestion
    parent_dir = './ImageToSketch/sketches'
```